lang/go/go.talon

```diff
@@ -1,195 +1,37 @@
 
-state while: "while "
-while <user.text> [over]:
-    insert("while ")
-    insert(user.formatted_text(text, "PRIVATE_CAMEL_CASE"))
-
-state for: "for "
-for <user.text> [over]:
-    insert("for ")
-    insert(user.formatted_text(text, "PRIVATE_CAMEL_CASE"))
-
-state for range: "forr "
-range <user.text> [over]:
-    insert("forr ")
-    insert(user.formatted_text(text, "PRIVATE_CAMEL_CASE"))
-
-state format: "fmt"
-format <user.text> [over]:
-    insert("fmt.")
-    insert(user.formatted_text(text, "PUBLIC_CAMEL_CASE"))
-
-state switch: "switch "
-switch <user.text> [over]:
-    insert("switch ")
-    insert(user.formatted_text(text, "PRIVATE_CAMEL_CASE"))
-
-state select: "select "
-# "select <user.text>:insert("select "), insert(user.formatted_text(text, "PRIVATE_CAMEL_CASE")]
-state (const | constant): " const "
-constant <user.text> [over]:
-    insert("const ")
-    insert(user.formatted_text(text, "PUBLIC_CAMEL_CASE"))
-
-state case: " case "
-state default: " default:"
-case <user.text> [over]:
-    insert("case ")
-    insert(user.formatted_text(text, "PRIVATE_CAMEL_CASE"))
-
-state type: " type "
-type <user.text> [over]:
-    insert("type ")
-    insert(user.formatted_text(text, "PUBLIC_CAMEL_CASE"))
-state true: " true "
-state false: " false "
-state (start | struct | struck):
-    insert(" struct {")
-    key("enter")
-(struct | struck) <user.text> [over]:
-    insert(" struct {")
-    key("enter")
-    insert(user.formatted_text(text, "PUBLIC_CAMEL_CASE"))
-
-[state] empty interface: " interface{} "
-state interface:
-    insert(" interface {")
-    key("enter")
-interface <user.text> [over]:
-    insert(" interface {")
-    key("enter")
-    insert(user.formatted_text(text, "PUBLIC_CAMEL_CASE"))
-
-state string: " string "
-[state] (int | integer | ant): "int"
-state slice: " []"
-slice of: "[]"
-[state] (no | nil): "nil"
-state (int | integer | ant) sixty four: " int64 "
-state tag: user.insert_between(" `", "`")
-field tag <user.text> [over]:
-    user.insert_between(" `", "`")
-    sleep(100ms)
-    insert(user.formatted_text(text, "snake"))
-    insert(" ")
-    sleep(100ms)
-
-state return: " return "
-return <user.text> [over]:
-    insert("return ")
-    insert(user.formatted_text(text, "PRIVATE_CAMEL_CASE"))
-
-map of string to string: " map[string]string "
-map of <user.text> [over]:
-    insert("map[")
-    insert(user.formatted_text(text, "PRIVATE_CAMEL_CASE"))
-    key("right")
-    sleep(100ms)
-
-receive: " <- "
-make: "make("
-loggers [<user.text>] [over]:
-    insert("logrus.")
-    insert(user.formatted_text(text, "PUBLIC_CAMEL_CASE"))
-
-length <user.text> [over]:
-    insert("len(")
-    insert(user.formatted_text(text, "PRIVATE_CAMEL_CASE"))
-
-append <user.text> [over]:
-    insert("append(")
-    insert(user.formatted_text(text, "PRIVATE_CAMEL_CASE"))
+[state] context: " ctx "
+[state] context (are you | argue): " ctx context.Context "
+state any: " any "
 
 state (air | err): "err"
```

Comment:
I find `err` and `error` to both have high misrecognition rates. This is more idiosyncratic, but I use `oops` for `err` and `boom` for `error`. Not sure I can fairly foist those on others.